feathr_project/feathr/client.py

```diff
@@ -439,15 +449,15 @@ def _materialize_features_with_config(self, feature_gen_conf_path: str = 'featur
         Args
           feature_gen_conf_path: Relative path to the feature generation config you want to materialize.
         """
-        _FeatureRegistry.save_to_feature_config(Path("./"))
-        if not feature_gen_conf_path.startswith('feature_gen_conf'):
-            raise RuntimeError(
-                'Feature generation config should be in feature_gen_conf folder.')
+        # _FeatureRegistry.save_to_feature_config(Path("./"))
```

Comment:
I think we should still save it, but to a different path?

Reply:
Added it back